Source/WebCore/css/process-css-properties.py

```diff
@@ -4411,7 +4415,7 @@ def generate_call_string(self, *, range_string, context_string):
                     return f"{builtin.consume_function_name}({range_string}, {context_string}, {builtin.mode}, {builtin.value_range}, {builtin.unitless})"
                 return f"{builtin.consume_function_name}({range_string}, {context_string}, {builtin.value_range}, {builtin.unitless})"
             elif isinstance(builtin, BuiltinLengthPercentageConsumer):
-                return f"{builtin.consume_function_name}({range_string}, {context_string}, {builtin.value_range}, {builtin.unitless})"
+                return f"{builtin.consume_function_name}({range_string}, {context_string}, {builtin.value_range}, {builtin.unitless}, {builtin.unitless_zero}, {builtin.negative_percentage}, {builtin.anchor}, {builtin.anchor_size})"
```

Comment:
as discussed we should do this for anchor() too.

Reply:
I forgot that, will make a bugzilla.

Reply:
https://bugs.webkit.org/show_bug.cgi?id=280797